client/components/checkbox/style.scss

```diff
@@ -0,0 +1,52 @@
+.checkbox {
+	cursor: pointer;
+	display: block;
+	font-size: 14px;
+	margin-bottom: 5px;
+	position: relative;
+	user-select: none;
+}
+
+.checkbox input[type="checkbox"] {
+	@include hide-content-accessibly;
+	border: 0;
+	margin: -1px;
```

Comment:
TBH, I don't know, I copied the styles for the `componentes/forms/checkbox` and I presumed that it was for something, so I preferred to keep it.